README.md

```diff
@@ -13,7 +13,8 @@ Python library with command line interface for building docker images.
  * build inside a docker container (so your builds are separated between each other)
  * git as a source to your Dockerfile (you may specify commit/branch and path to Dockerfile within the git repo)
  * collect build logs
- * integration with [koji](http://koji.fedoraproject.org/koji/) build system
+ * integration with
+   [koji](https://github.com/projectatomic/atomic-reactor/blob/master/docs/koji.md) build system
```

Comment:
Decided not to do this -- the relative links are pretty awkward-looking, and I'm not sure that they make any sense at all outside of the context of viewing the file on GitHub. Leaving them as full URIs allows the document to be read offline.